libs/design-system/src/panda/NovuPandaPreset.ts

```diff
@@ -27,16 +29,26 @@ export const NovuPandaPreset = definePreset({
     tokens: {
       sizes: SIZES_TOKENS,
       spacing: SPACING_TOKENS,
-      colors: COLOR_PALETTE_TOKENS,
+      colors: {
+        ...COLOR_PALETTE_TOKENS,
+        ...LEGACY_COLOR_TOKENS,
```

Comment:
what are your thoughts on the legacy tokens? maybe there is no sense in exposing them and just refactoring the old code to the new colors?

Reply:
I just added this last night because it will make our gradual adoption MUCH easier. Nikolay did not use the new values in the Information Architecture designs, which made things very difficult to implement. The way I did this, I'm excited that it should be a good user experience while keeping styles consistent:

- Devs just use the new semantic tokens (i.e. `surface.page`)
- The Preset has legacy overrides for the semantic tokens that point them to the legacy colors.
- Once we're ready to switch colors, we just get rid of the semantic-to-legacy mapping layer, and voila!

Reply:
awesome!